kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/extensions/FileUtils.java

```diff
@@ -5,11 +5,26 @@
 import java.io.IOException;
 import java.nio.charset.StandardCharsets;
 
+import static org.apache.kafka.common.utils.Utils.readFileAsString;
+
 
 public class FileUtils {
 
-    public static String getResourceAsString(String resourceFileName) throws IOException {
-        return IOUtils.resourceToString("/" + resourceFileName, StandardCharsets.UTF_8);
+    public static String getResourceAsString(String resourceFileName) {
+        try {
+            return IOUtils.resourceToString("/" + resourceFileName, StandardCharsets.UTF_8);
+        } catch (IOException e) {
+            e.printStackTrace();
+            return null;
```

Comment:
let's put a `@Nullable` annotation above the methods which can return nulls

Reply:
Added @Nullable annotation